ideafast/ideafast-patientapp
# Implement Support Screen & Markdown Support

- [x] Assigned a Reviewer to the **PR**.
- [x] Assigned a Project to the **PR**.

## What Changed and Why?

  1. Added Support for importing markdown files
  2. Implemented store state for language selector
  3. Added default state to the store for devices
  4. Designed Device component and used on the support page
  5. Added structure and use of importing and rendering the content of the markdown files inside SupportDoc.

*Note:* the content of the markdown files are currently rendered and not transformed as that's outside the scope of this PR, i.e. the raw text is shown and not marked-up HTML or such. 

## How to Review Changes?

Verify that the support screen mirrors the prior [adobe design](https://xd.adobe.com/view/d8cb1f48-e69d-4acb-b97d-9ec34c15f0b7-8d98/screen/cffc1d75-1100-4262-9da7-a9d2b552d308) and that the loading of markdown files is correct. To do that:

  1. Open the app and go to settings (Contributions->Cog) and change the language to whatever you want.
  1. Tap the Support tab button, then choose one of the devices. This will take you to a new screen and the raw content of the markdown file inside. Depending on the langage and device chosen, the content should be "DeviceName LangCode"
 1. Tap back to the "Contributions" tab and the Settings screen will still be open. Change your language and tab back to "Support" and you'll see that the other language file has been loaded
